docs/src/test-cli-js.md

```diff
@@ -105,12 +59,330 @@ Complete set of Playwright Test options is available in the [configuration file]
+```
+
+#### Options
+
+| Option | Description |
+| :--- | :--- |
+| `--host <host>` | Host to serve report on (default: localhost) |
+| `--port <port>` | Port to serve report on (default: 9323) |
+
+#### Examples
+
+```bash
+# Show latest test report
+npx playwright show-report
+
+# Show a specific report
+npx playwright show-report playwright-report/
+
+# Show report on custom port
+npx playwright show-report --port 8080
+```
+
+### Install Browsers
+
+Install browsers required by Playwright.
+
+#### Syntax
+
+```bash
+npx playwright install [options] [browser...]
+npx playwright install-deps [options] [browser...]
+npx playwright uninstall
+```
+
+#### Install Options
+
+| Option | Description |
+| :--- | :--- |
+| `--force` | Force reinstall of stable browser channels |
+| `--with-deps` | Install browser system dependencies |
+| `--dry-run` | Don't perform installation, just print information |
+| `--only-shell` | Only install chromium-headless-shell instead of full Chromium |
+| `--no-shell` | Don't install chromium-headless-shell |
+
+#### Install Deps Options
+
+| Option | Description |
+| :--- | :--- |
+| `--dry-run` | Don't perform installation, just print information |
+
+#### Examples
+
+```bash
+# Install all browsers
+npx playwright install
+
+# Install only Chromium
+npx playwright install chromium
+
+# Install specific browsers
+npx playwright install chromium webkit
+
+# Install browsers with dependencies
+npx playwright install --with-deps
+```
+
+## Generation & Debugging Tools
+
+### Code Generation
+
+Record actions and generate tests for multiple languages. [Read more about Codegen](./codegen-intro.md).
+
+#### Syntax
+
+```bash
+npx playwright codegen [options] [url]
+```
+
+#### Options
+
+| Option | Description |
+| :--- | :--- |
+| `-b, --browser <name>` | Browser to use: chromium, firefox, or webkit (default: chromium) |
+| `-o, --output <file>` | Output file for the generated script |
+| `--target <language>` | Language to use: javascript, playwright-test, python, etc. |
+| `--test-id-attribute <attr>` | Attribute to use for test IDs |
+
+#### Examples
+
+```bash
+# Start recording with interactive UI
+npx playwright codegen
+
+# Record on specific site
+npx playwright codegen https://playwright.dev
+
+# Generate Python code
+npx playwright codegen --target=python
+```
+
+### Trace Viewer
+
+Analyze and view test traces for debugging. [Read more about Trace Viewer](./trace-viewer.md).
+
+#### Syntax
+
+```bash
+npx playwright show-trace [options] [trace...]
+```
+
+#### Options
+
+| Option | Description |
+| :--- | :--- |
+| `-b, --browser <name>` | Browser to use: chromium, firefox, or webkit (default: chromium) |
+| `-h, --host <host>` | Host to serve trace on |
+| `-p, --port <port>` | Port to serve trace on |
+
+#### Examples
+
+```bash
+# View a trace file
+npx playwright show-trace trace.zip
+
+# View trace from directory
+npx playwright show-trace trace/
+```
+
+## Specialized Commands
```

Comment:
I don't think any of the commands below bring any value here, and they will eventually diverge from the source of truth. I'd remove them.